src/NuGetTrends.Web/Portal/package.json

```diff
@@ -24,8 +24,8 @@
     "@angular/router": "^15.1.5",
     "@sentry-internal/feedback": "^0.0.1-alpha.17",
     "@sentry-internal/rrweb": "2.5.0",
```

Comment:
should we bump this to 2.6.0 too?
actually do we need this here explicitly?

Reply:
oh I'll bump it to 2.6.0!